Source/WebCore/platform/graphics/gstreamer/eme/CDMOpenCDM.cpp

```diff
@@ -323,11 +317,8 @@ void CDMInstanceOpenCDM::loadSession(LicenseType, const String& sessionId, const
     SessionLoadFailure sessionFailure(SessionLoadFailure::None);
 
     if (!session->load(responseMessage)) {
-        std::string request = "message:";
-        if (!responseMessage.compare(0, request.length(), request.c_str())) {
-            size_t length = messageLength(responseMessage, request);
-            GST_TRACE("message length %u", length);
-            auto message = SharedBuffer::create(responseMessage.c_str() + length, responseMessage.length() - length);
+        if (!responseMessage.compare(0, 8, "message:")) {
```

Comment:
Might want to factor this out as a check of "isMessage", it's used multiple times.

Reply:
This seems to be outdated